src/lib/y2storage/encryption.rb

```diff
@@ -79,6 +81,23 @@ def in_etc?
       in_etc_crypttab?
     end
 
+    # Returns the absolute path to a file containing the encryption password
+    #
+    # To be used as a third field in the crypttab file
+    #
+    # @return [String]
+    def password_file
+      @password_file
```

Comment:
you mention that it returns string, but in fact it can also return nil. So allow to return nil in documentation or ensure that nil cannot be returned.

Reply:
Ok. I will pay greater attention to this when update it.